cmd/ical-notifier/config.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/jm-lemmi/ical-relay/helpers"
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v3"
+)
+
+// STRUCTS
+// !! breaking changes need to keep the old version in legacyconfig.go !!
```

Comment:
we dont need that warning since its a new config with no previous structure

Reply:
ich finds fine das stehen zu lassen als hinweis, wenn die legacyconfig vllt wieder eingeführt wird oder von Tim wieder ins leben gerufen.